src/main/java/org/jabref/gui/Main.css

```diff
@@ -161,24 +348,54 @@
 .tab-pane > .tab-header-area > .headers-region > .tab {
     -fx-background-insets: 0;
     -fx-background-radius: 0;
-    -fx-background-color: -fx-accented-background;
-    -fx-border-color: -fx-border;
+    /*-fx-background-color: green;*/
+    /*-fx-background-color: -fx-accented-background;*/
+    -fx-border-color: -fx-outer-border;
     -fx-border-width: 0.5 0.5 0.5 0.5;
-    -fx-padding: 0.4em 0.9em 0.4em 0.9em;
+    -fx-padding: 0.3em 0.9em 0.3em 0.9em;
+    -fx-pref-height: -jr-header-height;
+
 }
 
 .tab-pane > .tab-header-area > .headers-region > .tab .tab-label {
-    -fx-text-fill: -fx-unimportant;
+    -fx-text-fill: -fx-mid-text-color;
+}
+
+.tab-pane > .tab-header-area > .headers-region > .control-buttons-tab {
+    -fx-border-color: -jr-theme;
+    -fx-fill: -jr-theme;
+    -fx-text-fill: -jr-theme;
 }
 
 .tab-pane > .tab-header-area > .headers-region > .tab:selected {
-    -fx-background-color: -fx-light-background;
-    -fx-border-color: -fx-active-background;
+    /*-fx-background-color: -fx-control-inner-background;*/
+    -fx-border-color: -jr-theme;
```

Comment:
I think it makes sense to introduce two new colors `-jr-accent` and `jr-accent-text` and use them for all selected/active states.